applications/ConstitutiveLawsApplication/custom_utilities/advanced_constitutive_law_utilities.h

```diff
@@ -476,5 +477,22 @@ class KRATOS_API(CONSTITUTIVE_LAWS_APPLICATION) AdvancedConstitutiveLawUtilities
     static double MacaullyBrackets(
         const double Number);
```

Comment:
```suggestion
    static double MacaullyBrackets(const double Number);
```
Style guide (one argument methods must be in one line).